### ISSUE TITLE
[feature request] Support STX transfers in deployments

### ISSUE THREAD

**Issue by hstove** [hirosystems/clarinet#492]
I've become a huge fan of Clarinet's "deployments" feature - both for actual deployments, but also for ad-hoc contract calls and BTC transfers. One thing I'd love to see is support for a `stx_transfer` transaction type. It's the only remaining thing I use a script for.

Assumed type:

```rust
pub struct StxTransferSpecificationFile {
  pub expected_sender: String,
  pub recipient: String,
  pub amount_ustx: u64,
  pub memo: Option<String>, // dont really think its useful for dev needs, but might as well?
  pub cost: u64,
}
```

**Comment by hstove** [comment]
Awesome! This can be closed, right?

**Comment by lgalabru** [comment]
Shipped in `v1.0.5`, so yes!
